web_shortcut/static/src/js/web_shortcut.js

```diff
@@ -6,6 +6,7 @@ odoo.define('web.shortcut', function (require) {
         WebClient = require('web.WebClient'),
         ViewManager = require('web.ViewManager'),
         ActionManager = require('web.ActionManager'),
+        Menu = require('web.Menu'),
```

Comment:
Was it required to add web.Menu to fix those bugs? Can you point me the 'reintroduced' bug by this revert?